lib/register.dart

```diff
@@ -0,0 +1,306 @@
+import 'package:InstiComplaints/modals.dart';
+import 'package:flutter/material.dart';
+import 'dart:math';
+
+class BackgroundMaker extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        width: MediaQuery.of(context).size.width,
+        height: MediaQuery.of(context).size.height,
+        child: Stack(
+          fit: StackFit.expand,
+          children: <Widget>[
+            Container(
+              constraints: BoxConstraints.expand(),
+              color: Colors.white,
+              child: RegisterPage(),
+            ),
+            ClipPath(
+                clipper: CurveClipper(),
+                child: Container(
+                  constraints: BoxConstraints.expand(),
+                  color: Color(0xFF181D3D),
+                  child: Column(children: [
+                    SizedBox(height: MediaQuery.of(context).size.height / 8),
+                    Text(
+                      'Sign Up',
+                      style: Theme.of(context)
+                          .textTheme
+                          .headline5
+                          .apply(color: Colors.white),
+                    )
+                  ]),
+                )),
+          ],
+        ));
+  }
+}
+
+class CurveClipper extends CustomClipper<Path> {
```

Comment:
CurveClipper is a custom widget which extends the CustomClipper widget of the type Path. In this widget, the getClip method is used to return the path i.e. the borders of the coloured region to clip this region. The Path is defined by making lines and arcs as per the requirement.